### DISCUSSION TITLE
Error on exportJob

### DISCUSSION THREAD

**Discussion by alanski66** [trendyminds/craft-reporter#3] [Q&A]
Had a quick run thru and cant get past this:
Trying to get property 'id' of non-object in /Users/site/craft/vendor/trendyminds/craft-reporter/src/jobs/ExportJob.php:79

Seems to be getting the logged in users id to store with new asset which exists.

`yii\base\ErrorException: Trying to get property 'id' of non-object in /Users/sites/craft/vendor/trendyminds/craft-reporter/src/jobs/ExportJob.php:79 Stack trace: #0 /Users/sites/craft/vendor/trendyminds/craft-reporter/src/jobs/ExportJob.php(79): yii\base\ErrorHandler->handleError(8, 'Trying to get p...', '/Users/sites...', 79, Array) #1 /Users/sites/craft/vendor/yiisoft/yii2-queue/src/Queue.php(246): trendyminds\reporter\jobs\ExportJob->execute(Object(craft\queue\Queue)) #2 /Users/sites/craft/vendor/yiisoft/yii2-queue/src/cli/Queue.php(162): yii\queue\Queue->handleMessage('21', 'O:35:"trendymin...', '300', '1') #3 /Users/sites/craft/vendor/yiisoft/yii2-queue/src/cli/Command.php(146): yii\queue\cli\Queue->execute('21', 'O:35:"trendymin...', '300', '1', '7888') #4 [internal function]: yii\queue\cli\Command->actionExec('21', '300', '1', '7888') #5 /Users/sites/craft/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array) #6 /Users/sites/craft/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array) #7 /Users/sites/craft/vendor/yiisoft/yii2/console/Controller.php(182): yii\base\Controller->runAction('exec', Array) #8 /Users/sites/craft/vendor/yiisoft/yii2/base/Module.php(552): yii\console\Controller->runAction('exec', Array) #9 /Users/sites/craft/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('queue/exec', Array) #10 /Users/sites/craft/vendor/craftcms/cms/src/console/Application.php(89): yii\console\Application->runAction('queue/exec', Array) #11 /Users/sites/craft/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('queue/exec', Array) #12 /Users/sites/craft/vendor/yiisoft/yii2/base/Application.php(384): yii\console\Application->handleRequest(Object(craft\console\Request)) #13 /Users/sites/craft/craft(23): yii\base\Application->run() #14 {main}`
Any ideas?

**Comment by chattervast** [comment]
Same issue. I change line 82 on vendor/trendyminds/craft-reporter/src/jobs/ExportJob.php to
`$asset->uploaderId = 1;`
this is a temporary fix as it doesnt seem able to get the user id

**Comment by aaronbushnell** [comment]
Hi, all—I'm pretty late on this but are you still having issues here? We've made a handful of changes to improve the reliability of this!
